mmseg/models/utils/inverted_residual.py

```diff
@@ -0,0 +1,120 @@
+import torch.nn as nn
+import torch.utils.checkpoint as cp
+from mmcv.cnn import ConvModule
+
+from .se_layer import SELayer
+
+
+class InvertedResidual(nn.Module):
```

Comment:
There is also one `InvertedResidual` in `mmseg/model/utils/`. 
We should have a consistent one.

Reply:
I've merged these two "inverted residual" files. Now the final version of "InvertedResidual" is in mmseg/model/utils only.